src/main/resources/static/js/subscription.js

```diff
@@ -382,7 +391,7 @@ jQuery(document).ready(function () {
             "ajax": {
                 "url": addBackendParameter(frontendServiceUrl + "/subscriptions"),
                 "type": "GET",
-                "dataSrc": "",   // Flat structure from EI backend REST API
+                "dataSrc": "",   //Flat structure from EI backend REST API
```

Comment:
There seems to be differnt formatters in use here... Since this removes spaces in comment, but further down space is added in the comments. Not sure if it's automatic formatting but please keep it consistent.